SimFastTiming/FastTimingCommon/python/mtdDigitizer_cfi.py

```diff
@@ -58,15 +58,15 @@
         bxTime             = cms.double(25),
         etaResolution      = cms.string("0.03+0.0025*x"), # This is just a dummy dependence on eta.
         # n bits for the ADC 
-        adcNbits           = cms.uint32(12),
+        adcNbits           = cms.uint32(8),
         # n bits for the TDC
-        tdcNbits           = cms.uint32(12),
+        tdcNbits           = cms.uint32(11),
         # ADC saturation
-        adcSaturation_MIP  = cms.double(102),
+        adcSaturation_MIP  = cms.double(25),
```

Comment:
do I understand correctly that changes like this mean that this PR introduces changes at the digitization step?
Please decouple these from the hit finder and track refitter updates for appropriate separation of physics features.